src/components/Shared/Filters/FilterListItem.tsx

```diff
@@ -1,57 +1,75 @@
 import { ListItem, ListItemText } from '@material-ui/core';
 import React from 'react';
-import { Filter } from './Filter';
+import { ContactFilterValue } from '../../Contacts/ContactFilters/ContactFilters';
+import {
+  Filter,
+  CheckboxFilter,
+  DaterangeFilter,
+  MultiselectFilter,
+  NumericRangeFilter,
+  RadioFilter,
+  TextFilter,
+  DateRangeInput,
+  NumericRangeInput,
+} from '../../../../graphql/types.generated';
 import { FilterListItemCheckbox } from './FilterListItemCheckbox';
 import { FilterListItemDateRange } from './FilterListItemDateRange';
 import { FilterListItemMultiselect } from './FilterListItemMultiselect';
+import { FilterListItemNumericRange } from './FilterListItemNumericRange';
 import { FilterListItemSelect } from './FilterListItemSelect';
 import { FilterListItemTextField } from './FilterListItemTextField';
 
 interface Props {
   filter: Filter;
-  value?: boolean | string | Array<string>;
-  onUpdate: (value?: boolean | string | Array<string>) => void;
+  value?: ContactFilterValue;
+  onUpdate: (value?: boolean | string | Array<string> | DateRangeInput) => void;
```

Comment:
Is this the right type? Should it be this?
```suggestion
  onUpdate: (value?: ContactFilterValue) => void;
```